index.html

```diff
@@ -64,5 +63,14 @@
 
     <script src="https://js.stripe.com/v3/"></script>
     <script src="https://www.google.com/recaptcha/api.js?onload=vueRecaptchaApiLoaded&render=explicit" async defer></script>
+    <script>
+      (function () {
+        if (window.origin === 'http://localhost:8080') {
+          let scriptEl = document.createElement('script')
+          scriptEl.setAttribute('src', 'http://localhost:8098')
```

Comment:
Is this a script for Algolia or for testing?

Reply:
I put it in for testing